hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/Quota.java

```diff
@@ -75,8 +75,10 @@ public Quota(Router router, RouterRpcServer server) {
    * @param storagespaceQuota Storage space quota.
    * @param type StorageType that the space quota is intended to be set on.
    * @param checkMountEntry whether to check the path is a mount entry.
-   * @throws AccessControlException If the quota system is disabled or if
+   * @throws IOException If the quota system is disabled or if
    * checkMountEntry is true and the path is a mount entry.
+   * By the way, this latter case actually throws an AccessControlException,
+   * which happens to be a subclass of IOException.
```

Comment:
this isn't required I think,
below it throws ACE directly as well
```
    if (checkMountEntry && isMountEntry(path)) {
      throw new AccessControlException(
          "Permission denied: " + RouterRpcServer.getRemoteUser()
              + " is not allowed to change quota of " + path);
    }
```

Reply:
Ok, that's a little more concise. I have updated the PR, please check it again.